src/com/owncloud/android/datamodel/OCFile.java

```diff
@@ -456,6 +459,7 @@ private void resetData() {
         mIsDownloading = false;
         mEtagInConflict = null;
         mSharedWithSharee = false;
+        mPrivateLink = "";
```

Comment:
Why have you initialized this variable as "" instead of null, in the same way as mEtagInConflict variable?